src/backend/app/projects/project_routes.py

```diff
@@ -16,13 +27,74 @@
 from app.config import settings
 from app.users.user_deps import login_required
 from app.users.user_schemas import AuthUser
+from app.tasks import task_schemas
 
 router = APIRouter(
     prefix=f"{settings.API_PREFIX}/projects",
     responses={404: {"description": "Not found"}},
 )
 
 
+@router.get("/{project_id}/download-boundaries", tags=["Projects"])
+async def download_boundaries(
+    project_id: Annotated[
+        UUID,
+        Path(
+            description="The project ID in UUID format.",
+        ),
+    ],
+    db: Annotated[Connection, Depends(database.get_db)],
+    user_data: Annotated[AuthUser, Depends(login_required)],
+    task_id: Optional[UUID] = Query(
+        default=None,
+        description="The task ID in UUID format. If not provided, all tasks will be downloaded.",
+    ),
+    split_area: bool = Query(
```

Comment:
What does this do ?

Reply:
> What does this do ?

The `split_area` parameter is used to control the type of geometry returned in the downloaded GeoJSON file:

- **When `split_area` is `True`:**  
  The GeoJSON file will contain the boundaries of individual tasks within a project. These tasks are typically smaller subdivisions of the overall project area, and each one will be represented as a separate polygon. The resulting GeoJSON will be a `FeatureCollection` where each feature corresponds to a task boundary.
![Screenshot from 2024-09-03 13-12-50](https://github.com/user-attachments/assets/4be1469c-b2ae-4104-a77c-e21752b46a85)

- **When `split_area` is `False`:**  
  The GeoJSON file will contain the entire Area of Interest (AOI) as a single geometry. 
![Screenshot from 2024-09-03 13-12-11](https://github.com/user-attachments/assets/58112c2c-2382-4b7d-953d-af6ef01969fb)